README.md

```diff
@@ -140,6 +140,31 @@ Thus, **lincheck** can test that the testing data structure is correct even with
 This sequential specification class should have the same methods as the testing data structure.
 The specification class can be defined via `sequentialSpecification` parameter in both `Options` instances and the corresponding annotations.
 
+## Validation functions
+It is possible in *lincheck* to add the testing data structure invariants validation,
+which is implemented via functions that can be executed multiple times during execution
+when there is no running operation in an intermediate state (e.g., in the stress mode
+they are invoked after each of the init and post part operations and after the whole parallel part).
+Thus, these functions should not modify the data structure.
+
+Validation functions should be marked with `@Validate` annotation, do not have arguments,
```

Comment:
Just mess with tenses:
Validation functions should be marked ... have no arguments and should not return anything